src/subgradients.jl

```diff
@@ -21,10 +21,10 @@ for ``c\neq 1`` or ``p\neq  q``. Note that for the remaining case ``c=1``,
 
 # Optional
 
-* `c` – (`1`) the exponent of the distance,  i.e. the default is the distance
+* `c` – (`2`) the exponent of the distance,  i.e. the default is the distance
 * `atol` – (`0`) the tolerance to use when evaluating the distance between `p` and `q`.
 """
-function subgrad_distance(M, q, p, c::Int = 2; atol = 0)
+function subgrad_distance(M, q, p, c::Int = 2; atol = zero(eltype(first(p))))
```

Comment:
```suggestion
function subgrad_distance(M, q, p, c::Int = 2; atol = zero(number_eltype(p)))
```
`number_eltype` is more generic.

Reply:
Oh that's very nice!